magprime/models.py

```diff
@@ -12,6 +12,24 @@ def invalid_notification(self):
             except:
                 log.error('unable to send invalid email')
 
+    @cost_property
+    def child_discount(self):
+        if 'val' in self.age_group_conf and self.age_group_conf['val'] == c.UNDER_13:
```

Comment:
The check for 'val' in `age_group_conf` shouldn't generally be needed, but our unit tests tripped over this otherwise.